src/nvim/ops.c

```diff
@@ -332,8 +332,9 @@ static void shift_block(oparg_T *oap, int amount)
   const int oldstate = State;
   char_u *newp;
   const int oldcol = curwin->w_cursor.col;
-  const int p_sw = get_sw_value(curbuf);
-  const int p_ts = (int)curbuf->b_p_ts;
+  int p_sw = (int)get_sw_value_indent(curbuf);
+  long *p_vts = curbuf->b_p_vts_array;
+  const long p_ts = curbuf->b_p_ts;
```

Comment:
This is not in the patch.

Reply:
As mentioned, I moved some variables from int to long to remove compiler warnings. This variable was declared just above as `int` before. I'll remove the ordering change to make it clearer.